packages/angular/projects/ui-angular/src/lib/components/authenticator/components/amplify-authenticator/amplify-authenticator.component.html

```diff
@@ -54,99 +14,85 @@
         [active]="actorState?.matches('signIn')"
       >
         <!-- signIn component -->
-        <ng-container
-          [ngTemplateOutlet]="customComponents.signIn || signIn"
-          [ngTemplateOutletContext]="context"
-          *ngIf="actorState?.matches('signIn')"
-        ></ng-container>
+        <amplify-slot name="sign-in" *ngIf="actorState?.matches('signIn')">
+          <amplify-sign-in></amplify-sign-in>
+        </amplify-slot>
```

Comment:
No more of these repeated `ngTemplateOutlet` logics!

Reply:
Nice!